extension/src/test/e2e/wdio.conf.ts

```diff
@@ -33,7 +33,7 @@ export const config: Options.Testrunner = {
   capabilities: [
     {
       browserName: 'vscode',
-      browserVersion: 'insiders',
+      browserVersion: 'stable',
```

Comment:
Something has happened in insiders which means you cannot focus the plots webview after the experiments webview has been opened. I am still looking into it but this should get everything moving again.

Reply:
The problem is only for the test suite and `webdriverio`, there is no loss of functionality with insiders.